init.sh

```diff
@@ -2,138 +2,92 @@
 
 ok="Ok, going to the next question."
 
-cat << "EOF"
+read -r -d '' dna_ascii << "EOF"
 
 -. .-.   .-. .-.   .-. .-.   .
   \   \ /   \   \ /   \   \ /
  / \   \   / \   \   / \   \
 ~   `-~ `-`   `-~ `-`   `-~ `-
 
 EOF
+echo "${dna_ascii}"
+
 echo "Welcome! Before adding that folder (Floflis DNA) to your device, we need to configure the install, to make it more lightweight."
 echo ""
-echo "We will ask some questions and configure according to your answers. Please reply by typping a number (according to your choice) and press [ENTER]."
-cat << "EOF"
+echo "We will ask some questions and configure according to your answers. Please reply by typing y/n (according to your choice) and press [ENTER]."
 
--. .-.   .-. .-.   .-. .-.   .
-  \   \ /   \   \ /   \   \ /
- / \   \   / \   \   / \   \
-~   `-~ `-`   `-~ `-`   `-~ `-
+echo "${dna_ascii}"
```

Comment:
use "${var}" istead of "$var" for code readability and clarity

Reply:
Getting rid of brackets and commas/collons is why I prefer shell.
But, if this adds compatibility with other shells than BASH, ok.